data/hai/hai reveal 4 middle.txt

```diff
@@ -1413,13 +1413,122 @@ mission "Hai Reveal [A11] Sound the Alarm"
-			`	One of the Elders takes the time to address you as a witness, and confirm that threat was indeed real. Even so, though, that only seems to sway a small selection. One of the more stubborn Elders stands up. "The Unfettered have made serious attempts to push through at Wah Ki before, to no avail. More recently, they've been more trouble again, and again we have organized more than enough security to keep their disruption contained, even with the current human crisis. This is ridiculous, and the Unfettered would not dare, not while they are so dependent on our food and medical supplies these past years."`
-			`	"You are blind. Blind and old and foolish," snaps Teeneep. "If you will not even listen to this, then apparently I must now be the one to present a pathway to avoid war. If you do not take action, I will make the history of inaction I have discovered public knowledge. Every Hai from Greenwater to Snowfeather will know by the end of the day."`
-			`	Two different Elders respond simultaneously. The first says, "Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone."`
-			`	That one is left searching the room for someone who might actually be listening halfway through his sentence, though, as the second points at Teeneep and loudly says, "You wouldn't dare!" To which Teeneep responds, "It is already scheduled to happen unless I actively stop it!"`
-			`	Sayari has to raise her voice above the din and disentangles their two responses for you as the room devolves into shouting and several Elders who look quite worried leave the room. A moment later all three of you are escorted out and the chambers are closed.`
+			`Upon your return, Teeneep lodges a formal demand for another audience with the Elders, including the context that she has acquired support from the Unfettered with appropriate proof. Once sent, Sayari steps out for a while to talk to someone and ensure that the demand is being processed with appropriate speed, leaving you and Teeneep alone in her office.`
+			`	Teeneep spends a few moments more doing some other things at her terminal before finally sitting back with an expression of quiet determination. In the quiet she turns to you from her seat and says, "Well, all the pieces are set, now I suppose all that remains is to play out the moves."`
+			label teeneepquestions
+			choice
+				`	"What will happen now then?"`
```

Comment:
```suggestion
				`	"What will happen now, then?"`
```

Reply:
Technically correct, but I feel like it breaks up the flow when reading it as spoken text.

Reply:
Then we can keep it in.